handlers.go

```diff
@@ -1105,9 +1105,39 @@ func (s *Server) dealStatusByID(ctx context.Context, dealid uint) (*dealStatus,
 type getContentResponse struct {
 	Content      *Content       `json:"content"`
 	AggregatedIn *Content       `json:"aggregatedIn,omitempty"`
+	Selector     string         `json:"selector,omitempty"`
 	Deals        []*contentDeal `json:"deals"`
 }
 
+func (s *Server) calcSelector(aggregatedIn uint, contentID uint) (string, error) {
+	// sort the known content IDs aggregated in a CAR, and use the index in the sorted list
+	// to build the CAR sub-selector
+
+	// SELECT ordinal - 1 FROM (
+	//   SELECT
+	//     id, ROW_NUMBER() OVER ( ORDER BY CAST(id AS TEXT) ) AS ordinal
+	//   FROM contents
+	//   WHERE aggregated_in = ?
+    // ) subq
+	// WHERE id = ?
+
+	subQuery := s.DB.Table("contents").
+		Select("id, ROW_NUMBER() OVER ( ORDER BY CAST(id as TEXT) ) AS ordinal").
+		Where("aggregated_in = ?", aggregatedIn)
+
+	var ordinal uint
+	result := s.DB.Table("(?) subq", subQuery).
+		Select("ordinal - 1").
+		Where("id = ?", contentID).
+		Find(&ordinal)
```

Comment:
I think doing all of this through gorm is a mistake. Why not just ( as per https://gorm.io/docs/sql_builder.html#Raw-SQL )
*bad formatting!!!*
```suggestion
  var ordinal uint
  result := s.DB.Raw(`
	 SELECT ordinal - 1 FROM (
	   SELECT
	     id, ROW_NUMBER() OVER ( ORDER BY CAST(id AS TEXT) ) AS ordinal
	   FROM contents
	   WHERE aggregated_in = ?
     ) subq
	 WHERE id = ?
  `,aggregatedIn, contentID ).Scan(&ordinal)
```

Reply:
@marshall ping on ☝️ this?

Reply:
sorry for the late reply on this, agreed w/ this suggestion. will send the updated PR post-haste